projects/packages/my-jetpack/src/products/class-protect.php

```diff
@@ -116,4 +116,14 @@ public static function get_pricing_for_ui() {
 	public static function get_manage_url() {
 		return admin_url( 'admin.php?page=jetpack-protect' );
 	}
+
+	/**
+	 * Return product bundles list
+	 * that supports the product.
+	 *
+	 * @return boolean|array Products bundle list.
```

Comment:
```suggestion
	 * @return array Products bundle list.
```
Since this method always returns an array, I believe we can remove the `bool` return type.

Reply:
I think the force push re-introduced this.

Reply:
Re-introduced this again, keeps disappearing for some reason.